media/server/gstplayer/public/IGstPlayer.h

```diff
@@ -199,6 +199,23 @@ class IGstPlayer
      *
      */
     virtual void renderFrame() = 0;
+
+    /**
+     * @brief Set level and transition of audio attenuation.
+     *        Sets the current volume for the pipeline (0.0 silent -> 1.0 full volume)
+     *
+     * @param[in] volume Target volume level (0.0 - 1.0)
```

Comment:
Add ':' in param description here and in getVolume below.
Add retval.

Reply:
done